src/kvstore/RocksEngine.cpp

```diff
@@ -330,6 +330,7 @@ int32_t RocksEngine::totalPartsNum() {
 
 ResultCode RocksEngine::ingest(const std::vector<std::string>& files) {
     rocksdb::IngestExternalFileOptions options;
+    options.move_files = true;
```

Comment:
how about use a gflag here?

Reply:
> how about use a gflag here?

I agree with you, or add a parameter for ingest.